diff --git a/pyproject.toml b/pyproject.toml
index 22ff2d5..6f7afb7 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -11,17 +11,16 @@ dependencies = [
   "safetensors",
   "tensorboard",
   "tqdm",
-  "torch==2.6",
+  "torch==2.9.0",
   "triton>=3.2",
   # "git+https://git@github.com/kyutai-labs/moshi.git#egg=moshi&subdirectory=moshi",
   # We could also use [tool.uv.sources] to point to the git repo, but this maintains
   # compatibility with `pip install -e .`
-  "moshi @ git+https://github.com/kyutai-labs/moshi.git#subdirectory=moshi",
   "sphn==0.1.12",
   "auditok==0.2",
   "whisper_timestamped",
   "huggingface_hub",
-  "torchaudio",
+  "torchaudio==2.9.0",
   "submitit",
   "llvmlite>=0.44",
   "numba>=0.61",
