diff --git a/moshi/pyproject.toml b/moshi/pyproject.toml
index 766e694..4c80067 100644
--- a/moshi/pyproject.toml
+++ b/moshi/pyproject.toml
@@ -6,12 +6,11 @@ dependencies = [
     "numpy >= 1.26, < 2.3",
     "safetensors >= 0.4.0, < 0.6",
     "huggingface-hub >= 0.24, < 0.34",
-    "bitsandbytes >= 0.45, < 0.46; sys_platform == 'linux'",
     "einops >= 0.7, < 0.9",
     "sentencepiece == 0.2",
     "sounddevice == 0.5",
     "sphn >= 0.1.4, < 0.2.0",
-    "torch >= 2.2.0, < 2.8",
+    "torch >= 2.9.0, < 3.0",
     "aiohttp>=3.10.5, <3.12",
     "pytest >= 8.3.3",
 ]
